wikitexthtml/page.py

```diff
@@ -30,8 +31,10 @@ def __init__(self, page: str) -> None:
         self._errors = []  # type: List[str]
         self._template_path = []  # type: List[str]
 
-    def prepare(self, body) -> wikitextparser.WikiText:
-        body = preprocess.begin(self, body)
+    def prepare(self, body, should_color_syntax=False) -> wikitextparser.WikiText:
```

Comment:
`should_color_syntax` -> `enable_syntax_highlight`? `with_syntax_highlight`?

I find it a bit hard that sometimes you call it "color syntax" and other moments "syntax highlight". That makes it difficult to read the code.